test/errors.jl

```diff
@@ -92,12 +117,14 @@
     end
 
     @testset "UnsupportedAttribute" begin
-        @test_throws MOI.UnsupportedAttribute begin
-            MOI.set(model, MOI.ObjectiveFunction{MOI.SingleVariable}(),
-                     MOI.SingleVariable(vi))
-        end
-        @test_throws MOI.UnsupportedAttribute begin
-            MOI.set(model, MOI.ConstraintDualStart(), ci, 0.0)
+        for f in (MOI.set, MOIU.load, MOIU.allocate)
```

Comment:
Consider `@testset "Unsupported attribute with $f" for f in (MOI.set, MOIU.load, MOIU.allocate)`